Dockerfile

```
@@ -0,0 +1,15 @@
+FROM node:14.17.3-alpine
+
+ENV NODE_ENV=development
+
+WORKDIR /app
+
+COPY package.json ./
```

We can use this instead, as we will also be tracking the lock file as well.
```suggestion
COPY package*.json ./
```